package.json

```diff
@@ -145,7 +145,7 @@
     "@uniswap/sdk-core": "^3.0.1",
     "@uniswap/smart-order-router": "^2.10.0",
     "@uniswap/token-lists": "^1.0.0-beta.30",
-    "@uniswap/universal-router-sdk": "1.3.4",
+    "@uniswap/universal-router-sdk": "^1.3.6",
```

Comment:
Why `^`?

Reply:
every other package basically uses ^ and i don't see a downside, especially since we have a yarn.lock